nucypher/blockchain/eth/token.py

```diff
@@ -128,6 +128,9 @@ def __int__(self) -> int:
         """Cast to smallest denomination"""
         return int(self.to_nunits())
 
+    def __round__(self, decimals: int = 0):
```

Comment:
This is neat. Is this `__round__` specifically so you can call the built-in on this class?

Reply:
yes!